### ISSUE TITLE
allure-behave how to add a new information into file.json generate by allure-behave 

### ISSUE THREAD

**Issue by Khouloud-moalla** [allure-framework/allure2#1672]
Generate reports with allure behave: My Test will be executed on several browsers in parallel so when I generated the report i will have two reports but whithout the name of browser in file.json . What I want doing to know which report belongs to which browser !! Any heelp please

**Comment by baev** [comment]
Duplicate of #130
